### ISSUE TITLE
Doesn't process Relative path

### ISSUE THREAD

**Issue by cvax** [tomoki1207/gfm-preview#1]
If you try to add HTML code into your MarkDown, this previewer can't render it.

For example, if I use this
```
<a href="http://www.greenteapress.com/thinkpython/"><img src="img/think_python.png" style="width: 100px;" target="_blank"></a>
```
Instead of this:
```
[Python](img/think_python.png)
```
It just gives a missing/broken image icon in this previewer.

**Comment by tomoki1207** [comment]
@cvax Thank you for reporting.
It seems caused by relative path rather than html rendering.

The below was work fine,
```
<h1>Python</h1>
```

but the below was not work.
```
[Python](img/think_python.png)
```

**Comment by tomoki1207** [comment]
@cvax This issue fixed at v0.1.2.
Could you check it?

**Comment by cvax** [comment]
@tomoki1207 Thanks. Looks good.
